data/release/global.xml

```diff
@@ -4,9 +4,9 @@
     <addressCountry>Egypt</addressCountry>
     <addressType>Work</addressType>
     <contactEmail>claudius.ptolomaeus@gmail.com</contactEmail>
-    <contactOrganization>The ancient geographes INC</contactOrganization>
+    <contactOrganization>The Ancient Geographers</contactOrganization>
     <contactPerson>Claudius Ptolomaeus</contactPerson>
-    <contactPosition>Chief geographer</contactPosition>
+    <contactPosition>Anax Geographer</contactPosition>
```

Comment:
Hmm, [Anax](https://en.wikipedia.org/wiki/Anax) doesn't really [translate well](https://translate.google.com/?ie=UTF-8&hl=en&client=tw-ob#auto/en/%E1%BC%84%CE%BD%CE%B1%CE%BE) in Google. :) Let's stick with "Chief".